### DISCUSSION TITLE
Exporting your data

### DISCUSSION THREAD

**Discussion by wraix** [chromiecraft/chromiecraft#3768] [General]
Hi ChromieCraft

Thanks for a good community and a well run and fun server!

I would like to request the ability to export my data for import into my own instance of azerothcore. Would you be willing to support that in the spirit of open source?

Kind Regards,
Wraix

**Comment by heyitsbench** [comment]
Given the existence of the .pdump command in typical AzerothCore, I would greatly appreciate this feature being available on ChromieCraft.
